packages/smb/src/core/index.tsx

```diff
@@ -17,6 +17,7 @@ window.reapit = {
     googleAnalyticsKey: '',
     cognitoOAuthUrl: '',
     cognitoUserPoolId: '',
+    accessToken: '',
```

Comment:
@willmcvay @tanphamhaiduong for now, we cannot access the `window.reapit.config` inside worker thread so I need to pass the access token and graphql backend uri from main thread to work thread. 
I think we should put access token inside the config object because if we save it on localstorage we still need the config object to get the graphql server uri.